### PR TITLE
account+view pages updated and account page added

### PR DESCRIPTION
**Pull request by ma-zah**
pls review

### CODE REVIEW

File: account-view/test-reports.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  />
+</Frame>
+
+There will be a row for each test with the following information:
+
+| column      | description                                               |
+| ----------- | --------------------------------------------------------- |
+| description | name of the test case                                     |
+| status ✅   | execution successful                                      |
+| status ⌛️  | execution in progress                                     |
```
Comment (ma-zah) [inline account-view/test-reports.mdx:47]:
```suggestion
| status ⌛️ | execution in progress|
```

File: edit-test-case.mdx [hunk 2]
```diff
@@ -101,11 +101,31 @@ Click on the `refresh` icon next to the existing **test dependency** and select
...
-`Run` you test case to see if it works. If it's successful, you can click on the `save` button. We will warn you if you try to leave without saving the test.
+`Save & run` you test case to see if it works. You can click on the `run only` button if you want to validate the test first before saving it.
+
+Btw, we will warn you if you try to leave without saving the test case.
 
 <Tip>
-  Double check that your newly edited step is `on` in the test case view if you
-  want to run it in your next **test report**.
+  Double check that your newly edited test is active - the toggle is `on`, if
+  you want to run it in your next **test report**.
```
Comment (fabianhinz) [inline edit-test-case.mdx:112]: maybe `- the toggle is on. Those active test cases will be executed in your next test report` ?

**Review by fabianhinz — APPROVED** [review]
only some small suggestions

File: account-view/projects.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+You can run your test **against a different URL** while keeping the subpaths - the URL is "templated". This allows you to [run your test cases against multiple deployments](/execute-test-cases#1-trigger-a-test-run) of the same application.
+
+<Frame caption="test target url of a project, 08/2024">
+  <img
+    src="images/accounts/test-target.png"
+    alt="test target url of a project"
+  />
+</Frame>
+
+Projects are listed in the left menu bar underneath your organization. The active project is shown in the top menu to the right
```
Comment (ma-zah) [inline account-view/projects.mdx:18]:
```suggestion
Projects are listed in the hamburger menu under your organization.
```

File: account-view/test-cases.mdx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Test cases"
+description: "This is where all your tests live"
+icon: "database"
+---
+
+A **test case** is a set of steps that mimic a user flow. You can find an overview of all your test cases in the `test case` section of the `project overview` page or in the `test cases` page accessible from the left sidebar.
+
+It is the place where you can add, edit or delete test cases. From here, you can also grow your test suite by having our AI agent suggest new tests based on specific test cases.
```
Comment (ma-zah) [inline account-view/test-cases.mdx:9]:
```suggestion
It is the place where you can add, edit or delete test cases. From here, you can also grow your test suite by [having our AI agent suggest new tests](/new-test-case#have-ai-agent-suggest-and-auto-generate-more-tests) based on specific test cases.
```

File: account-view/test-reports.mdx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    src="images/accounts/test-report-list.png"
+    alt="test report list in the project overview in the ap, screenshot 08/2024"
+  />
+</Frame>
+
+## Scheduled test runs
+
+If you want to [run your tests on a fixed schedule](/scheduled-execution), the `test reports` section in the `project overview` gives you the option of a daily, weekly or bi-weekly interval.
+
+This ensures that your app works flawlessly over time.
```
Comment (ma-zah) [inline account-view/test-reports.mdx:24]:
```suggestion
This ensures that your app works flawlessly over time. We will send you an email notification if any of your tests fail.
```

File: edit-test-case.mdx [hunk 6]
```diff
@@ -101,11 +101,31 @@ Click on the `refresh` icon next to the existing **test dependency** and select
   />
 </Frame>
 
-## Run and save edits
+## Save and run edits
 
-`Run` you test case to see if it works. If it's successful, you can click on the `save` button. We will warn you if you try to leave without saving the test.
+`Save & run` you test case to see if it works. You can click on the `run only` button if you want to validate the test first before saving it.
```
Comment (ma-zah) [inline edit-test-case.mdx:106]:
```suggestion
`Save & run` your test case to see if it works. You can click on the `run only` button if you want to validate the test first before saving it.
```

File: edit-test-case.mdx [hunk 7]
```diff
@@ -101,11 +101,31 @@ Click on the `refresh` icon next to the existing **test dependency** and select
...
 
-`Run` you test case to see if it works. If it's successful, you can click on the `save` button. We will warn you if you try to leave without saving the test.
+`Save & run` your test case to see if it works. You can click on the `run only` button if you want to validate the test first before saving it.
+
+Btw, we will warn you if you try to leave without saving the test case.
 
 <Tip>
-  Double check that your newly edited step is `on` in the test case view if you
-  want to run it in your next **test report**.
+  Double check that your newly edited test is active - the toggle is `on`, if
```
Comment (ma-zah) [inline edit-test-case.mdx:111]:
```suggestion
  Double check that your newly edited test is active - the toggle is `on`. Your active
```

File: edit-test-case.mdx [hunk 8]
```diff
@@ -101,11 +101,31 @@ Click on the `refresh` icon next to the existing **test dependency** and select
...
-`Run` you test case to see if it works. If it's successful, you can click on the `save` button. We will warn you if you try to leave without saving the test.
+`Save & run` your test case to see if it works. You can click on the `run only` button if you want to validate the test first before saving it.
+
+Btw, we will warn you if you try to leave without saving the test case.
 
 <Tip>
-  Double check that your newly edited step is `on` in the test case view if you
-  want to run it in your next **test report**.
+  Double check that your newly edited test is active - the toggle is `on`. Your active
+  you want to run it in your next **test report**.
```
Comment (ma-zah) [inline edit-test-case.mdx:112]:
```suggestion
tests will run it in your next **test report**.
```

File: edit-test-case.mdx [hunk 9]
```diff
@@ -101,11 +101,31 @@ Click on the `refresh` icon next to the existing **test dependency** and select
...
-`Run` you test case to see if it works. If it's successful, you can click on the `save` button. We will warn you if you try to leave without saving the test.
+`Save & run` your test case to see if it works. You can click on the `run only` button if you want to validate the test first before saving it.
+
+Btw, we will warn you if you try to leave without saving the test case.
 
 <Tip>
-  Double check that your newly edited step is `on` in the test case view if you
-  want to run it in your next **test report**.
+  Double check that your newly edited test is active - the toggle is `on`. Your active
+tests will run it in your next **test report**.
```
Comment (ma-zah) [inline edit-test-case.mdx:112]:
```suggestion
  active tests will run it in your next **test report**.
```

**Pull request merged**